contributors.js

```diff
@@ -75,4 +75,5 @@ const contributors = [
   { username: 'Yeashu', message: 'Hello Hacktoberfest. Here my journey begin'},
   { username: 'sanaa-duhh', message: 'starting my hacktoberfest journey with this one'},
   { username: 'SayedZahur786', message: 'Hello Zenika, Thanks for this Pr'}
```

Comment:
there is another thing to before accepting your PR, fix the json file adding a comma please

Reply:
done